client/src/modules/users/templates/user.name.cell.html

```diff
@@ -0,0 +1,3 @@
+<div class="ui-grid-cell-contents text-muted" ng-class="{'strike' : row.entity.deactivated }">
```

Comment:
It looks like `text-muted` will always be applied to this cell. I think it's probably better to only apply it on deactivation, similar to how our buttons work.

```html
<div class="ui-grid-cell-contents" ng-class="{'strike  text-muted' : row.entity.deactivated }">

```